src/emqx_channel.erl

```diff
@@ -1531,11 +1531,26 @@ init_alias_maximum(_ConnPkt, _ClientInfo) -> undefined.
 %%--------------------------------------------------------------------
 %% Enrich Keepalive
 
+%% MQTT 5
+ensure_keepalive(_AckProps, #channel{conninfo = #{keepalive := DefaultKeepAlive,
```

Comment:
The logic in this place should preferentially fetch from the 'Server-Keep-Alive' property of AckProps. as it may be set back by the 'client.connect' hook.

'client.connect' should be the highest priority
followed by the emqx.conf configuration item
and lastly the client's request for